src/main/java/cnLabs/Micro/clientapplication/Model/CustomUserDetails.java

```diff
@@ -0,0 +1,50 @@
+package cnLabs.Micro.clientapplication.Model;
+
+import lombok.*;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.security.core.userdetails.UserDetails;
+
+import java.util.Collection;
+import java.util.List;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class CustomUserDetails implements UserDetails {
+
+//    @JsonProperty("id")
+    private Long id;
+    private String username;
+    private String password;
+    private String email;
+    private String fullName;
+
+    @Override
+    public Collection<? extends GrantedAuthority> getAuthorities() {
+        List<GrantedAuthority> authorities = List.of(new SimpleGrantedAuthority("ROLE_USER"));
+        return authorities;
+    }
+
+    @Override
+    public boolean isAccountNonExpired() {
+        return true;
+    }
+
+    @Override
+    public boolean isAccountNonLocked() {
+        return true;
+    }
+
+    @Override
+    public boolean isCredentialsNonExpired() {
+        return true;
+    }
+
+    @Override
+    public boolean isEnabled() {
+        return true;
+    }
```

Comment:
Hardcoding security is not good :) 

In the production system the `users` service would have `CustomUserDetails` would have fields like:
```java
    private boolean accountExpired;
    private boolean accountLocked;
    private boolean credentialsExpired;
    private boolean enabled;
```
and the methods would return actual values, or they would have some logic to get the values.

If the system would set to use the credentials for security, other services in the system would request user entry from `users` service. info would come in a DTO that will have the mentioned private fields with values.